cfme/infrastructure/virtual_machines.py

```diff
@@ -151,6 +267,48 @@ class InfraVmReconfigureView(BaseLoggedInPage):
     is_displayed = False
 
 
+class MigrateView(BaseLoggedInPage):
+    title = Text('#explorer_title_text')
+
+    @View.nested
+    class form(View):  # noqa
+        submit = Button('Submit')
+        cancel = Button('Cancel')
+
+        @View.nested
+        class request(Tab):  # noqa
+            TAB_NAME = 'Request'
```

Comment:
**REQUIRED**
`TAB NAME` is only needed when a class name cannot be converted to a tab name with capitalize method.

Reply:
it was used all over provisioning view - so just sticking to common practice